app/services/processors/blog_post_views_updater.rb

```diff
@@ -14,6 +14,8 @@ def call
           create_metric(year, month, views)
         end
       end
+    rescue Faraday::Error => exception
```

Comment:
doesn't ExceptionHunter track exceptions automatically? 🤔 cc @brunvez

Reply:
yeah it should automatically track exceptions on jobs and if they are not handled on every request

Reply:
It certainly does! But it doesn't if we catch the error first 😅